pdfkit.gemspec

```diff
@@ -13,8 +13,6 @@ Gem::Specification.new do |s|
   s.description = "Uses wkhtmltopdf to create PDFs using HTML"
   s.license     = "MIT"
 
-  s.rubyforge_project = "pdfkit"
-
```

Comment:
This is deprecated and soon removed:
```
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
```

Reply:
👍